v2/car.go

```diff
@@ -17,9 +22,9 @@ var (
 		0x02, // uint(2)
 	}
 	// The size of the CAR v2 prefix in 11 bytes, (i.e. 11).
-	PrefixBytesSize = len(PrefixBytes)
+	PrefixBytesSize = uint64(len(PrefixBytes))
```

Comment:
I also think this is a bit redundant; `len(PrefixBytes)` is self-explanatory and easy to type. The user can also choose what integer type they want to convert to.

Reply:
We could but then would it make sense to get the length of a constant byte prefix and cast it to uint64 on every offset calculation?

Reply:
If you prefer to have this variable for your own implementation, I think that's fine, but I don't think you need to expose it publicly. Anything that gets made public has to stay there forever, and makes the godoc index larger.